cmd/kubeadm/app/cmd/init.go

```diff
@@ -169,9 +168,34 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight
 	// Try to start the kubelet service in case it's inactive
 	preflight.TryStartKubelet()
 
-	return &Init{cfg: cfg}, nil
+	// validate version argument
```

Comment:
please remove this, it's a bad rebase. no code should be added before `return &Init{cfg: cfg}, nil` here